src/designer/Designer.ts

```diff
@@ -150,10 +150,10 @@ export class Designer {
      * @param {number} boardId
      * @returns {Record<string, FormV1>} an object of id and FormResponse
      */
-    getForms(boardId: number): Promise<Record<string, FormV1>> {
+    getForms(boardId: number, selectedFormId?: number): Promise<Record<string, FormV1>> {
         return new Promise((resolve, reject) => {
             this.fetcher
-                .get<Record<string, FormV1>>(`api/v1/boards/${boardId}/forms`)
+                .get<Record<string, FormV1>>(`api/v1/boards/${boardId}/forms`, { params: { form_id: selectedFormId } })
```

Comment:
I'm commented that in the Folloe repo, but I think it should have been a separate GET forms/${formId}